runtime/jcl/common/vm_scar.c

```diff
@@ -364,7 +362,7 @@ J9VMDllMain(J9JavaVM *vm, IDATA stage, void *reserved)
 			break;
 
 		case ALL_VM_ARGS_CONSUMED :
-			FIND_AND_CONSUME_VMARG(STARTSWITH_MATCH, VMOPT_XJCL_COLON, NULL);
+			FIND_AND_CONSUME_VMARG(STARTSWITH_MATCH, "-Xjcl:", NULL);
```

Comment:
We could remove this for jdk25+.

Reply:
I could do that, but then the test _would_ need to be updated. Is that your preference?